File: FPYJS-4/task2/prime.js
console.time()
input = process.argv[2]

function isPrime(num) {
    for (let i = 2; i * i <= num; i++)
        if (num % i === 0)
            return false;
    return num > 1;
}

function primeArr(input) {
    let arr = []
    for (let i = 2; i <= input; i++) {
        if (isPrime(i) === true) {
            arr.push(i);
        }
    }
    return arr
}

console.log(primeArr(input));
console.timeEnd()

File: FPYJS-4/task2/prime.py
import sys
import time


start_time = time.time()
input = int(sys.argv[1])

def is_prime(num):
    for i in range(2, num):
        if i * i <= num:
            if num % i == 0:
                return False
            i += 1
    return num > 1

def prime_arr(input):
    arr = []
    [arr.append(i) for i in range(2, input+1) if is_prime(i) is True]
    return arr

print(prime_arr(input))
print(f'{(time.time() - start_time):.3f} seconds')
